javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/MetricServer.java

```diff
@@ -26,7 +26,7 @@
    * @param producer The metric producer
    * @param config The configuration
    */
-  void start(MetricProducer producer, Properties config);
+  void start(MetricProducer producer, Config config);
```

Comment:
I don't know if anyone is using this (I hope not) but let's wait to break this SPI until #2442 also

Reply:
@trask did you mean to ask @breedx-splk to revert this change as well?